lib/operations.coffee

```diff
@@ -108,6 +108,15 @@ exports.execute = (image, operations, options) ->
 
 	emitter = new EventEmitter()
 
+	# There is an edge case where the event emitter instance
+	# emits the `end` event before the client is able to
+	# register a listener for it.
+	emitterOn = emitter.on
```

Comment:
am I right it's only used once below? can be `emitter.on(arguments...)`, so why plucking the `on` method here?